src/doctypes/contacts.js

```diff
@@ -0,0 +1 @@
+export const CONTACTS_DOCTYPE = 'io.cozy.contacts'
```

Comment:
:thinking: shouldn't we put those doctypes constants in a single file? Case where there is more than 1 constant per file seems to be rare